util/util.go

```diff
@@ -45,3 +65,52 @@ func ParseWeekdays(weekdays string) []time.Weekday {
 	}
 	return parsedWeekdays
 }
+
+// ParseTimePeriods takes a comma-separated list of time periods in Kitchen24 format and turns them
+// into a slice of TimePeriods. It ignores any whitespace.
+func ParseTimePeriods(timePeriods string) ([]TimePeriod, error) {
+	parsedTimePeriods := []TimePeriod{}
+
+	for _, tp := range strings.Split(timePeriods, ",") {
+		parts := strings.Split(tp, "-")
+		if len(parts) != 2 {
+			continue
+		}
+
+		begin, err := time.ParseInLocation(timeFormatKitchen24, strings.TrimSpace(parts[0]), time.Local)
```

Comment:
Don't you want to recognize the timezone set by the user in the config?

Reply:
Yeah, not sure what I did there. I think my intention was to just parse the string into an exactly corresponding time object regardless of any timezone, e.g. parsing a full time string "... 14:00 someTimezone" into an object with `hour=14`. I think `time.Parse` without `Location` will just do that?!

Reply:
Seems like this should work. Just be careful and thorough in your testing. Maybe use UTC just for consistency with your `TimeOfDay` function?